website/views.py

```diff
@@ -283,22 +284,21 @@ def dashboard(auth):
 @must_be_logged_in
 def watched_logs_get(**kwargs):
     user = kwargs['auth'].user
-    page_num = int(request.args.get('pageNum', '').strip('/') or 0)
-    page_size = 10
-    offset = page_num * page_size
-    recent_log_ids = itertools.islice(user.get_recent_log_ids(), offset, offset + page_size + 1)
+    page = int(request.args.get('page', '').strip('/') or 0)
```

Comment:
What if the client passes in invalid data? Invalid inputs should be handled by returning the proper error response to the user.